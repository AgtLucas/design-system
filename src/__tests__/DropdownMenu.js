import React from 'react'
import renderer from 'react-test-renderer'
import { DropdownMenu } from '..'

describe('DropdownMenu', () => {
  test('renders', () => {
    const json = renderer
      .create(<DropdownMenu>DropdownMenu Button 1</DropdownMenu>)
      .toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders "height" prop', () => {
    const json = renderer
      .create(<DropdownMenu height={50}>DropdownMenu Button 2</DropdownMenu>)
      .toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders "activeDescendantIndex" prop', () => {
    const json = renderer
      .create(
        <DropdownMenu activeDescendantIndex={1}>
          DropdownMenu Button 3
        </DropdownMenu>
      )
      .toJSON()
    expect(json).toMatchSnapshot()
  })
})
