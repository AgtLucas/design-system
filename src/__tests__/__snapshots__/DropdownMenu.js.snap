// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownMenu DropdownMenu renders 1`] = `
.c0 {
  border: 1px solid #d1d6db;
  border-radius: 2px;
  height: 200px;
  overflow: scroll;
  -webkit-overflow-scrolling: scroll;
}

<div
  aria-labelledby="dropdownButton"
  className="c0 "
>
  DropdownMenu Button
</div>
`;
